src/Clock.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+interface Props {
+  clockName: string;
+}
+
+interface State {
+  today: Date,
+  timerId: number,
+}
+
+export class Clock extends React.Component<Props> {
+  state: State = {
+    today: new Date(),
+    timerId: 0,
+  };
+
+  componentDidMount() {
+    this.state.timerId = window.setInterval(() => {
```

Comment:
You can't assign to state like this, use [this.setState](https://react.dev/reference/react/Component#setstate)
```suggestion
    this.state.timerId = window.setInterval(() => {
```